grr/server/grr_response_server/gui/ui/components/client_details/entry_history_dialog/entry_history_dialog.ts

```diff
@@ -28,7 +28,7 @@ interface EntryHistoryTableRow {
 })
 export class EntryHistoryDialog {
   readonly entryType: EntryType;
-  readonly tableRows: EntryHistoryTableRow[] = [];
+  readonly tableRows: EntryHistoryTableRow<any>[] = [];
```

Comment:
I see that looping through generics everywhere might be out of scope for this PR. Please add a comment that disables ts-lint's any detection for this line.

Reply:
Is it though? It is newly added code and I don't think there is a good reason not make it properly typed from the very beginning.

Here, the `EntryHistoryDialog` can hold only is expected to hold history of a particular type. It seems easy enough to make it generic as well.

Reply:
I agree it is not preferred to use `any` as type. In order to use a specific type for the array, we could use a generic type for the `EntryHistoryDialog`, just like you suggested. Unfortunately, Angular doesn't consider the generic type if we do that, and it would be of no use. I am not able to find a solution for this problem and if you or @max-vogler have any other suggestions, please reach out.

Reply:
Discussed offline. For now, seems like we have to leave it as it is.